### PR TITLE
Switching to modules

### PR DESCRIPTION
**Pull request by rgritti**

### CODE REVIEW

**Comment by danpersa** [comment]
@rgritti you need to update the documentation as well

File: UPGRADE.md [hunk 1]
```diff
@@ -8,14 +8,7 @@
 
 ```bash
 brew upgrade vips
-brew upgrade glide
 brew upgrade golang 
-glide update
```
Comment (danpersa) [inline UPGRADE.md]: @rgritti what's the go modules version of this? Is there a way to automatically update all dependencies?

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+require (
+	github.com/abbot/go-http-auth v0.0.0-20181006234207-98f2d47741b6
+	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973
+	github.com/cjoudrey/gluahttp v0.0.0-20180401075744-7065ce73b887
+	github.com/cjoudrey/gluaurl v0.0.0-20161028222611-31cbb9bef199
+	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd
+	github.com/dimfeld/httppath v0.0.0-20170720192232-ee938bf73598
+	github.com/gogo/protobuf v1.1.1
+	github.com/golang/protobuf v1.2.0
```
Comment (danpersa) [inline go.mod]: Why are there two versions of protobuf?

File: go.mod [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	github.com/prometheus/common v0.0.0-20181019103554-16b4535ad14a
+	github.com/prometheus/procfs v0.0.0-20181005140218-185b4288413d
+	github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a
+	github.com/sirupsen/logrus v1.0.4
+	github.com/sony/gobreaker v0.0.0-20180905101324-b2a34562d02c
+	github.com/szuecs/rate-limit-buffer v0.7.0
+	github.com/uber/jaeger-client-go v2.14.0+incompatible
+	github.com/uber/jaeger-lib v1.5.0
+	github.com/yuin/gopher-lua v0.0.0-20181008131706-f6fcaab0c612
+	github.com/zalando/skipper v0.10.100
```
Comment (danpersa) [inline go.mod]: the latest version of skipper is 1.10.126? would it be possible to upgrade?

File: UPGRADE.md (hunk 1 above)
Comment (rgritti) [reply UPGRADE.md]: there isn't.
When you run `go test ./..` of `go build ./...` it will first update the dependencies

File: go.mod (hunk 3 above)
Comment (rgritti) [reply go.mod]: v0.10.126 is the latest release: https://github.com/zalando/skipper/releases

**Comment by danpersa** [comment]
👍

**Comment by danpersa** [comment]
👍

**Pull request merged**